scclasses/SCVim.sc

```diff
@@ -155,8 +156,24 @@ classvar <scvim_dir, <scvim_cache_dir,
         };
 
         postln("SCVim: processing help docs, this takes a little while....");
+        // regexp searching for filenames
+        glob_files = { arg folder, match;
+                var p;
+                PathName(folder).filesDo { arg x;
+                        if (match.matchRegexp(x.fileName)) {
+                                p = p.add(x.fileNameWithoutExtension);
+                                p = p.add(x.fullPath);
+                        }
+                };
+                p
+        };
+        postln("SCVim: Searching for all help files in " ++ Platform.resourceDir);
+        all_help_files =  Dictionary.newFrom(glob_files.value(Platform.resourceDir, "(?i)\\.(html|htm|rtf|schelp|scd)$"));
```

Comment:
@gogobd *.rtf help files are gone since quite some time now so I'd rather not add any new code supporting them. I might as well just add a commit to your PR..